src/app/checkout/navigateToOrderConfirmation.tsx

```diff
@@ -2,10 +2,10 @@ import { noop } from 'lodash';
 
 import { isBuyNowCart } from '../common/utility';
 
-export default function navigateToOrderConfirmation(location = window.location, orderId?: number): Promise<never> {
+export default function navigateToOrderConfirmation(location = window.location, orderId?: number, isBuyNowCartEnabled?: boolean): Promise<never> {
     let url: string;
 
-    if (orderId && isBuyNowCart()) {
+    if (orderId && isBuyNowCartEnabled && isBuyNowCart()) {
```

Comment:
I am thinking does it make sense to split this method into two, one for if the buyNowcart experiment is enabled and another for the present workflow?

Reply:
I have wrapped up `isBuyNowCartEnabled` for simple removal as we'll remove it eventually.